core/config/locales/en.yml

```diff
@@ -1086,6 +1086,7 @@ en:
     cannot_create_payment_without_payment_methods_html: You cannot create a payment
       for an order without any payment methods defined. %{link}
     cannot_create_returns: Cannot create returns as this order has no shipped units.
+    cannot_edit_completed_orders: Cannot edit completed orders.
```

Comment:
Not sure what wording we'd like to display here, but after submitting this I'm thinking it should be more broad than "completed" orders. Perhaps something more like `Cannot edit existing orders.` or `You may only edit your current shopping cart.`

Reply:
`Cannot edit placed orders`, maybe?

Reply:
@dportalesr I changed to `You may only edit your current shopping cart.` which I think is most appropriate. It doesn't matter what id you enter you'll see your current cart. We also don't want to indicate to people what existing order numbers are in our system or what state they happen to be in.